lib/tasks/tasks.rake

```diff
@@ -76,7 +77,12 @@ def execute(instance, sequences)
       end
     end
     instance.save!
-    sequence_instance = sequence.new(instance, client, false)
+
+    # manually read config yaml file
+    config = YAML.load_file(ENV['INFERNO_CONFIG_FILE'] || 'config.yml')
+    disable_verify_peer = config['disable_verify_peer']
```

Comment:
Manually reading config file seems brittle.  I did some searching around how to leverage the work Sinatra is already doing here and came across a [SO question/answer with an example](https://stackoverflow.com/questions/36507763/how-to-access-sinatra-custom-settings-from-service) which led me to:

```
Inferno::App::Endpoint.settings.disable_verify_peer
```
That seems to work for me.  Have you tried this?